packages/vm/tsconfig.json

```diff
@@ -1,4 +1,5 @@
 {
   "extends": "../../config/tsconfig.json",
   "include": ["src/**/*.ts", "src/**/*.json", "tests/**/*.ts", ],
+  "compilerOptions": {"skipLibCheck": true}
```

Comment:
What is this option doing?

Reply:
It skips node_modules during `tsc` so that errors from installed libraries don't show up.